packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed an issue where packageName could result in an improper file path.
```

Comment:
can we be a little more specific on what the issue is? i think we should mention this is a fix for the spec reporter, regarding packages within mono repos

Reply:
little more specific here https://github.com/cerner/terra-toolkit/pull/617/commits/6b8f2b8d59fce9ff47c70329be39d2968fb4c3cf#diff-5121be21da689ee0a1bd5a529bbd578dc2bde92ce93b497f118e58290be63dc1R6